### PR TITLE
Devops/terraform/internal route

### PR DESCRIPTION
**Pull request by sfawcett123**
Added Internal Routes to allow for Prometheous to scrape application when metrics are in place.

Multiple routes are not available via the terraform provider when the application.routes method is employed, so removed the route from the application, and attached them as routes in the routes.tf file.
The CDN route remains unaffected as this is a special case.

Additionally since the test environment is no longer beta, the routes have been renamed accordingly, this has a knock on effect of impacting the statuscake tests

### CODE REVIEW

File: terraform/statuscake/test.env.tfvars [hunk 1]
```diff
@@ -9,4 +9,24 @@ ftt = {
...
+    website_name = "Get Into Teaching Website (Staging)"
+    website_url   = "https://get-into-teaching-app-test.london.cloudapps.digital/healthcheck.json"
+    test_type     = "HTTP"
+    check_rate    = 60
+    contact_group = [185037]
+    trigger_rate  = 0
+    custom_header = "{\"Content-Type\": \"application/x-www-form-urlencoded\"}"
+    status_codes  = "204, 205, 206, 303, 400, 401, 403, 404, 405, 406, 408, 410, 413, 444, 429, 494, 495, 496, 499, 500, 501, 502, 503, 504, 505, 506, 507, 508, 509, 510, 511, 521, 522, 523, 524, 520, 598, 599"
+  }
+rtt = {
```
Comment (jebw) [inline terraform/statuscake/test.env.tfvars]: These look like variable names? If so can they be renamed something more obvious for someone not already familiar with the project?

**Review by jebw — CHANGES_REQUESTED** [review]
Looks good - variable names could be more descriptive is the only concern.

File: terraform/statuscake/test.env.tfvars (hunk 1 above)
Comment (sfawcett123) [reply terraform/statuscake/test.env.tfvars]: Variables renamed

**Review by jebw — APPROVED** [review]
Perfect - makes much more sense to me as the layman :+1:

**Pull request merged**